assignment1/fc_net.py

```diff
@@ -54,12 +54,30 @@ def L_model_backward(self, AL, Y, caches):
         """
         Backward propagation process for the entire network.
 
-        :param AL:
-        :param Y:
-        :param caches:
-        :return:
+        :param AL: the probabilities vector, the output of the forward propagation (L_model_forward)
+        :param Y: the true labels vector (the "ground truth" - true classifications)
+        :param caches: list of caches containing for each layer: a) the linear cache; b) the activation cache
+        :return: a dictionary with the gradients
         """
-        pass
+
+        grads = []
+
+        last_layer_dA = -(Y/AL) + ((1-Y)/1-AL)
```

Comment:
that is cross entropy grad? 
maybe add comment?

Reply:
Look in the Lecture_1.pptx slide 50. i'll add a comment.

Reply:
Okay, I don't understand it. That is the loss gradient?  what is the loss in lecture 1?
 By the way, It should be (1-Y)/(1-AL)